### PR TITLE
Mex 358

### PR DESCRIPTION
**Pull request by dragos-rebegea**
## Reasoning
- update abi & view names
- add votinPower compute on microservice level

### CODE REVIEW

File: src/modules/governance/services/governance.abi.service.ts [hunk 1]
```diff
@@ -383,6 +391,23 @@ export class GovernanceTokenSnapshotAbiService
...
+        return new BigNumber(quorum).toFixed();
+    }
+
+    @ErrorLoggerAsync({ className: GovernanceTokenSnapshotAbiService.name })
+    @GetOrSetCache({
+        baseKey: 'governance',
+        remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
+        localTtl: CacheTtlInfo.ContractState.localTtl,
+    })
+    async votingPowerDecimals(scAddress: string): Promise<number> {
```
Comment (claudiulataretu) [inline src/modules/governance/services/governance.abi.service.ts:405]: you can move this method in governance.service.ts

**Review by claudiulataretu — CHANGES_REQUESTED** [review]

File: src/modules/governance/services/governance.abi.service.ts [hunk 2]
```diff
@@ -461,6 +489,18 @@ export class GovernanceEnergyAbiService
         return response.firstValue.valueOf().bech32();
     }
 
+    @ErrorLoggerAsync({ className: GovernanceEnergyAbiService.name })
+    @GetOrSetCache({
+        baseKey: 'governance',
+        remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
+        localTtl: CacheTtlInfo.ContractState.localTtl,
+    })
+    async userVotingPower(contractAddress: string, proposalId: number, userAddress: string) {
```
[the remark this answers is not in the document]
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:498]: done

File: src/modules/governance/services/governance.abi.service.ts (hunk 1 above)
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:405]: i would keep it here so we can have caching on it

File: src/modules/governance/services/governance.abi.service.ts [hunk 3]
```diff
@@ -129,8 +135,9 @@ export class GovernanceTokenSnapshotAbiService
         remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
         localTtl: CacheTtlInfo.ContractState.localTtl,
     })
-    async feeTokenId(scAddress: string): Promise<string> {
-        return await this.feeTokenIdRaw(scAddress);
+    async feeTokenId(scAddress: string): Promise<EsdtToken> {
```
[the remark this answers is not in the document]
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:138]: reverted

File: src/modules/governance/services/governance.abi.service.ts (hunk 1 above)
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts]: done

**Review by claudiulataretu — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Pull request merged**